### PR TITLE
[pota-quantization-value-test] Add tests for mixed-precision

### PR DESCRIPTION
**Pull request by jinevening**
This adds tests for mixed-precision quantization.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/8596
Draft PR: https://github.com/Samsung/ONE/pull/8656

### CODE REVIEW

File: compiler/pota-quantization-value-test/test.lst [hunk 1]
```diff
@@ -31,3 +31,32 @@ addTest(Split_000 channel int16)
...
+addQConfTest(Mul_001 layer uint8)
+addQConfTest(Mul_001 channel int16)
+addQConfTest(PRelu_001 layer uint8)
+addQConfTest(PRelu_001 channel int16)
+addQConfTest(ReLU_000 layer uint8)
+addQConfTest(ReLU_000 channel int16)
+#addQConfTest(Split_000 channel uint8)  Revisit when mixed-precision for multi-output Ops is supported.
+#addQConfTest(Split_000 channel int16)  Revisit when mixed-precision for multi-output Ops is supported.
+addQConfTest(TransposeConv_001 channel int16)
+addQConfTest(TransposeConv_001 layer uint8)
```
Comment (jinevening) [inline compiler/pota-quantization-value-test/test.lst:62]: These are added tests.

**Comment by jinevening** [comment]
Test data was copied from existing `pota-quantization-value-test`. For example,

Test data for "default u8 quant + s16 for the target Op" -> Use existing s16 golden data.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**